ClimsoftVer4/ClimsoftVer4/ucrValueFlagPeriod.vb

```diff
@@ -249,7 +249,7 @@ Public Class ucrValueFlagPeriod
 
     End Sub
 
-    Private Sub ucrValue_evtValueChanged(sender As Object, e As EventArgs) Handles ucrValue.evtValueChanged
+    Private Sub ucrValue_Leave(sender As Object, e As EventArgs) Handles ucrValue.Leave
```

Comment:
If i had a no value on the value textbox and M on the Flag, and then I do put in the value in the assign to all obs box, will this run? 
DO the controls get focus when we do the assign to all obs? Is not then this will be a bit of trouble because the controls will not get focus hence the leave event will not run.

Reply:
@Patowhiz I have tested this and we want to have `evtValueChanged` not `leave`